selectrum.el

```diff
@@ -1779,7 +1779,8 @@ defaults to `completion-extra-properties'."
 CANDIDATES is the list of candidate strings. DEFAULT is the default
 value which can be overridden and BUF the buffer the session was
 started from."
-  (setq-local selectrum-is-active t)
```

Comment:
It is actually there on purpose in case someone set it to nil in their setup hooks :laughing: But maybe we should trust our users more?